Dispatcher/Views/ArticleCell.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  ArticleCell.swift
+//  Dispatcher
+//
+//  Created by Ofek Cohen on 19/04/2023.
+//
+
+import UIKit
+
+//MARK: - ArticleCellDelegate
+
+protocol ArticleCellDelegate {
+    func navigateButtonPressed(_ articleTitle: String)
+    func favoritesButtonPressed(_ articleTitle: String)
+    
+}
+
+//MARK: - rticleCell class
+
+class ArticleCell: UITableViewCell {
+    
+    static let articleCellIdentifier = "articleCell"
+    
+
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var titleLabel: UILabel!
+    @IBOutlet weak var tagLabel: UILabel!
+    @IBOutlet weak var autherLabel: UILabel!
+    @IBOutlet weak var subTitleLabel: UILabel!
+    @IBOutlet weak var dispatchButton: UIButton!
+    @IBOutlet weak var moreTagsLabel: UILabel!
+    @IBOutlet weak var favoritesButton: UIButton!
+    
+    var delegate: ArticleCellDelegate?
+    
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        
+        makeCardShape()
+        setUIobjectsColor()
+        setUIobjectsShape()
+    }
+    
+    private func setUIobjectsShape() {
+        tagLabel.layer.cornerRadius = 10
+        moreTagsLabel.layer.cornerRadius = 10
+        dispatchButton.layer.cornerRadius = 20
+        favoritesButton.layer.cornerRadius = favoritesButton.frame.width / 2
+        favoritesButton.contentEdgeInsets = UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5)
+        
+    }
+    
+    private func setUIobjectsColor() {
+        tagLabel.textColor = UIColor(named: "labelsTextColor")
+        moreTagsLabel.textColor = UIColor(named: "labelsTextColor")
+        titleLabel.textColor = .black
+        subTitleLabel.textColor = UIColor(named: "labelsTextColor")
+        dateLabel.textColor = UIColor(named: "labelsTextColor")
+        autherLabel.textColor = UIColor(named: "labelsTextColor")
+        dispatchButton.setTitleColor(.white, for: .normal)
+    }
+    
+    private func makeCardShape() {
```

Comment:
Let's give the function a more meaningful name so it's easier to understand what's inside it (clean code rules).
We can name it something like "setCellMarginsAndBorder"